hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootControllerTest.java

```diff
@@ -364,7 +365,6 @@ public void rootRsIpAddressNotStoredIfDisabled() throws Exception {
             @Expect(type = TargetAssignDistributionSetEvent.class, count = 1),
             @Expect(type = ActionCreatedEvent.class, count = 1), @Expect(type = ActionUpdatedEvent.class, count = 1),
             @Expect(type = TargetUpdatedEvent.class, count = 2),
-            @Expect(type = TargetAttributesRequestedEvent.class, count = 1),
```

Comment:
Why there isn't a `TargetAttributesRequestedEvent` thrown anymore? Don't get me wrong, I'm just wondering why there is a behaviour change. Was there indeed an event triggered although action was closed with error or by the "success" statement afterwards?

Reply:
the deploymentActionFeedback method in line 381 did send "failure" as" feedback message" and not as "finished status", so the "finished status" was "success" and TargetAttributesRequestedEvent was thrown. Now the feedback is "closed with failure" which does not throw a TargetAttributesRequestedEvent. I think before the deploymentActionFeedback method was not used as intended.